### ISSUE TITLE
RFE: List available Node Images

### ISSUE THREAD

**Issue by christianh814** [kubernetes-sigs/kind#3053]
**What would you like to be added**:

A subcommand that lists the available node images. Something like `kind get node-images` or maybe another subcommand like `kind list node-images`. Possibly limit the return with `--limit` switch.

Looks like the API is there: https://hub.docker.com/v2/namespaces/kindest/repositories/node/tags

**Why is this needed**:

It is sometimes difficult/annoying to know what the "latest and greatest" tag for a specific version is. It would be cool if you can search for this directly in the CLI.

**Comment by BenTheElder** [comment]
There's multiple other open issues discussing variations on this and how we might improve image builds and tagging.

TLDR: we don't have a way for such a command to know if they were actually compatible, and it's difficult to guarantee that there's no issues without extensively testing.

The images that we expect to be compatible are published in the release notes for each version. Other tags are not given any such guarantee.

**Comment by BenTheElder** [comment]
This is however something we're considering with https://github.com/kubernetes-sigs/kind/issues/1895#issuecomment-1468991168

**Comment by BenTheElder** [comment]
/triage duplicate
https://github.com/kubernetes-sigs/kind/issues/2349
